packages/react-query/src/index.ts

```diff
@@ -1 +1,4 @@
+const suspensiveReactQueryVersion = '4'
```

Comment:
This was added to check the version of `@suspensive/react-query` being used in the user's environment. It is used to verify and guide the version through the CLI.